x-pack/test/functional/apps/ml/data_frame_analytics/trained_models.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const ml = getService('ml');
+
+  describe('trained models', function () {
+    before(async () => {
+      await ml.trainedModels.createdTestTrainedModels('classification', 15);
+      await ml.trainedModels.createdTestTrainedModels('regression', 15);
+      await ml.securityUI.loginAsMlPowerUser();
+      await ml.navigation.navigateToTrainedModels();
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
```

Comment:
We should probably also clean up the created models.

Reply:
I think the models are actually stored in one of the `.ml-` indices, so the existing cleanup works fine. Apologies for the noise.

Reply:
I've just double checked: the current cleanup also removes the created models :heavy_check_mark: